src/components/RichLayout/Header.js

```diff
@@ -1,87 +1,94 @@
+  /**
+   * @returns {boolean} True, if there's a location available with events or extras enabled.
+   */
+  isMenuEnabled () {
+    const location = this.getCurrentLocation()
+    return !!location && (location.extrasEnabled || location.eventsEnabled)
+  }
+
+  isExtrasEnabled () {
+    const location = this.getCurrentLocation()
+    return location && location.extrasEnabled
+  }
+
+  isEventsEnabled () {
+    const location = this.getCurrentLocation()
+    return location && location.eventsEnabled
   }
 
   render () {
+    const {t} = this.props
     return (
-      <header className={style.spacer}>
+      <Headroom>
         <div className={style.header}>
-          {/* Logo */}
-          <NavElement to={this.props.navigation.home} className={style.logo} disableActiveStyle={true}>
-            <img className={style.logoWide}
-                 src={logoWide}/>
-            <img className={style.logoSquare}
-                 src={logoSquare}/>
-          </NavElement>
-          <div className={style.itemsContainer}>
-            {/* Home for small devices */}
-            <NavElement to={this.props.navigation.home} className={cx(style.item, style.itemHome)}>
-              <FontAwesome className={style.fontAwesome} name='home'/>
-            </NavElement>
-            {/* Location */}
-            {this.props.location &&
-            <NavElement to={this.props.navigation.search} className={cx(style.item, style.itemSearch)}>
-              <FontAwesome className={style.fontAwesome} name='search'/>
-            </NavElement>
+          <div className={style.logoWide}>
+            <img src={logoWide}/>
+          </div>
+            <div className={style.actionItems}>
+            {
+              this.props.location &&
+              <Link href={this.props.navigation.search} className={style.actionItem}><img src={searchIcon}/></Link>
             }
-            <NavElement to={this.props.navigation.locationSelection} className={cx(style.item, style.itemLocation)}>
-              <FontAwesome className={style.fontAwesome} name='map-marker'/>
-            </NavElement>
-            {/* Language */}
-            {this.props.location &&
-            <HeaderDropDown className={style.itemLanguage} fontAwesome="language">
-              <LanguageFlyout />
+            <Link href={'/'} className={style.actionItem}><img src={locationIcon}/></Link>
+            <HeaderDropDown iconSrc={languageIcon}>
+              <LanguageFlyout/>
             </HeaderDropDown>
-            }
           </div>
+          {
+            this.isMenuEnabled() &&
+            <div className={style.menuItems}>
+              { this.isExtrasEnabled() && <Link href={this.props.navigation.home}>{t('common:extras')}</Link> }
+              <Link href={this.props.navigation.home}>{t('common:categories')}</Link>
+              { this.isEventsEnabled() && <Link href={this.props.navigation.events}>{t('common:news')}</Link> }
+            </div>
+          }
         </div>
```

Comment:
I need to have some logic, to detect wether Categories, Extras or Events is currently shown, but I am unsure how to do it. Do I really need to get router.path and check the url? 🤔

Reply:
https://github.com/FormidableLabs/redux-little-router#link
You can do:
<Link
  href='/wat'
  className='normal-link' 
  activeProps={{ className: 'active-link' }}
>
  Wat
</Link>`

:)
`

Reply:
I think the active class will only take effect, if the href *exactly* matches the current url (so if the url is `/augsburg/de/welcome`, then the active-class will be removed, but I still want the `Categories` to be printed bold.) But I'll give it a try...